### DISCUSSION TITLE
Third site/location for the Monitor

### DISCUSSION THREAD

**Discussion by allanharry** [hapostgres/pg_auto_failover#841] [Q&A] [answered]
Hi

I have been using/testing pg_auto_failover since early versions and extremely happy of the simplicity/robustness to enable HA.
In my company we have implemented pg_auto_failover on multiple clusters where some having 2 standby's to ensure sync replication.
We have a third datacenter in another country (low latency) and I have this question:
**Would it be considered to be the best possible solution to have the monitor running in a third datacenter in relation to split brain/network/quorum issues ?**

Regards Allan

**Comment by wilsoniya** [comment, marked as answer]
This is a great question and I'm interested in advice in this regard as well, thanks!
